assets/src/edit-story/components/library/common/index.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * Exports
+ */
+export { default as SearchInput } from './searchInput';
+export { default as Section } from './section';
+
+export const Title = styled.h3`
+  color: ${({ theme }) => theme.colors.fg.v1};
+  margin: 0;
+  font-size: 19px;
+  line-height: 1.4;
+  flex: 3 0 0;
+  display: flex;
+  align-items: center;
+  padding: 2px 0;
+`;
+
+export const Header = styled.div`
+  display: flex;
+  margin: 0 0 25px;
+`;
+
+export const MainButton = styled.button`
```

Comment:
The `UploadButton` in the media library should re-use this component. Right now `MediaLibrary` defines `buttonStyles` with basically the same styles and passes that to the button.

Also, on Figma the design seems to be a bit different for this (button that looks more like a link)

Reply:
I like this button a lot more as the main CTA, we'll take it to Sam

Reply:
@swissspidy could you please review the last commit for refactoring `UploadButton`? I'm still new to React hooks and I'm not sure if it's idiomatic to use a hook as a click action. What I'm looking for is something like an HOC.

Reply:
I like it! Seems intuitive to me 🙂